pyvista/plotting/theme.py

```diff
@@ -65,7 +65,8 @@
     'multi_rendering_splitting_position': None,
     'volume_mapper': 'fixed_point' if os.name == 'nt' else 'smart',
     'depth_peeling': {
-        'number_of_peels': 5,
+        'number_of_peels': 4,
+        'occlusion_ratio': 0.0,
```

Comment:
@GuillaumeFavelier, 0.0 is VTK's default for thee occlusion ratio and 4 is the default number of peels. I figured it would be good to go with those defaults since it makes #547 look really good.